app/src/main/kotlin/io/eugenethedev/taigamobile/data/repositories/Utils.kt

```diff
@@ -33,7 +33,8 @@ fun CommonTaskResponse.toCommonTask(commonTaskType: CommonTaskType) = CommonTask
     taskType = commonTaskType,
     colors = color?.let { listOf(it) } ?: epics.orEmpty().map { it.color },
     isClosed = is_closed,
-    tags = tags.orEmpty().map { Tag(name = it[0]!!, color = it[1].fixNullColor()) }
+    tags = tags.orEmpty().map { Tag(name = it[0]!!, color = it[1].fixNullColor()) },
+    blockedNote = blocked_note.takeIf { is_blocked }
 )
 
 fun String?.fixNullColor() = this ?: "#A9AABC" // gray, because api returns null instead of gray -_-
```

Comment:
Maybe move it to resources, 'cause I don't understand, that it's gray?

Reply:
It's been like this for a while, but still, that's a good point. But instead of resources I think it should be moved to the [Colors.kt](https://github.com/EugeneTheDev/TaigaMobile/blob/master/app/src/main/kotlin/io/eugenethedev/taigamobile/ui/theme/Colors.kt) (because we use Jetpack Compose and store colors as Kotlin variables)